scripts/generate_visualc_files.pl

```diff
@@ -178,6 +181,7 @@ sub gen_app {
 
 sub get_app_list {
     my $makefile_contents = slurp_file('programs/Makefile');
+    $makefile_contents =~ s/\$\(TF-PSA-CRYPTO_PROGRAM_PATH\)/..\/tf-psa-crypto\/programs/g;
```

Comment:
This line here is necessary to expand the path where `$(TF-PSA-CRYPTO_PROGRAM_PATH)` is used within the makefile otherwise tests cannot be found. If this script is going to be ported over to python, this might be easier to automate to enable tests to be moved without additional configuration of the scripts file.